Penumbra/Importer/TexToolsImport.cs

```diff
@@ -53,11 +53,8 @@ public void ImportModPack( FileInfo modPackFile )
             switch( modPackFile.Extension )
```

Comment:
why even really have this switch anymore if the version is used to check if its valid or not and what it is?